document/src/main/resources/config/liquibase/changelog/20240201-add-internal-case-status-table.xml

```diff
@@ -49,6 +49,13 @@
         </addColumn>
     </changeSet>
 
+    <changeSet id="2.5" author="Ritense">
+        <createIndex indexName="idx_internal_case_status_case_name_status_key" tableName="internal_case_status" unique="true">
+            <column name="case_definition_name"/>
+            <column name="internal_case_status_key"/>
+        </createIndex>
+    </changeSet>
+
     <changeSet author="Ritense" id="3">
         <addForeignKeyConstraint
```

Comment:
This changeset fails without an index on `referencedColumnNames="case_definition_name, internal_case_status_key"`